app/models/rule.rb

```diff
@@ -30,8 +30,8 @@ class Rule < ApplicationRecord
 
   has_many :profile_rules, dependent: :delete_all
   has_many :profiles, through: :profile_rules, source: :profile
-  has_many :rule_group_rules, dependent: :delete_all
-  has_many :rule_groups, through: :rule_group_rules
+  has_one :rule_group_rule, dependent: :delete
```

Comment:
Not sure about this, so I created https://issues.redhat.com/browse/RHICOMPL-3401

Reply:
Is this a duplicate of this ticket that I created? Maybe we can combine them? https://issues.redhat.com/browse/RHICOMPL-3384